lucene/analysis/common/src/test/org/apache/lucene/analysis/hunspell/TestPerformance.java

```diff
@@ -40,8 +42,15 @@
  * en.txt}) in a directory specified in {@code -Dhunspell.corpora=...}
  */
 @TestCaseOrdering(TestCaseOrdering.AlphabeticOrder.class)
-@Ignore("enable manually")
```

Comment:
I actually took advantage of this. Running all tests in a package skipped the ignored ones, and so the correctness was checked relatively quickly. Now that'd also include performance/allDict tests, and can become slower :( But I can work around that locally.

Reply:
Not if you create a launch config that just has one of those properties (just the hunspell.dictionaries) - then perf tests would be skipped. If you launch everything from the same launcher then indeed this will be a problem. I don't care much but it'd be my preference to control it somehow without modifying the code itself. You could use a custom property, test group annotation... whatever works. I think two properties already separate these tests well enough though.

Reply:
Sure, that's the way to go